jina/parsers/orchestrate/runtimes/remote.py

```diff
@@ -148,6 +148,24 @@ def mixin_gateway_parser(parser):
     )
 
 
+def mixin_gateway_protocol_parser(parser):
+    """Add the arguments for the protocol to the gateway parser
+
+    :param parser: the parser configure
+    """
+
+    from jina.enums import GatewayProtocolType
+
+    parser.add_argument(
+        '--protocol',
+        nargs='+',
```

Comment:
we need to have a plural alias

Reply:
What happens if I pass a single one?

Reply:
It will be read as a list of ports with 1 protocol. You can check the behavior here
tests/unit/parsers/peapods/runtimes/test_port_protocol_parser.py